Lib9c/Delegation/Delegatee.cs

```diff
@@ -452,6 +476,33 @@ private void TransferReward(T delegator, BigInteger share, LumpSumRewardsRecord
             }
         }
 
+        private void TransferReward(
+            T delegator,
+            BigInteger share,
+            RewardBase currentRewardBase,
+            RewardBase? lastRewardBase)
+        {
+            var currentCumulative = currentRewardBase.CumulativeRewardDuringPeriod(share);
+            var lastCumulative = lastRewardBase?.CumulativeRewardDuringPeriod(share);
+
+            foreach (var c in currentCumulative)
+            {
+                var lastCumulativeEach = lastCumulative?[c.Key] ?? c.Key * 0;
```

Comment:
There are too many `?`, I recommend a more readable code.

```suggestion
            var lastCumulative = lastRewardBase?.CumulativeRewardDuringPeriod(share)
                ?? ImmutableSortedDictionary<Currency, FungibleAssetValue>.Empty;

            foreach (var c in currentCumulative)
            {
                var lastCumulativeEach = lastCumulative.GetValueOrDefault(
                    c.Key, defaultValue: c.Key * 0);
```